test/unit/test-action.js

```diff
@@ -1529,6 +1529,19 @@ describes.fakeWin('Core events', {amp: true}, (env) => {
     );
   });
 
+  it('should trigger change event for <input type="file"> elements', () => {
+    const handler = window.document.addEventListener.getCall(3).args[1];
```

Comment:
Should we add a test for multiple files?

Reply:
I'm having trouble stubbing the `files` property since it should be set to a `FileList` and those [aren't easily constructable](https://stackoverflow.com/questions/52078853/is-it-posible-to-update-filelist/52079109). e2e is probably better here, thoughts?

Reply:
Maybe the `FileList` test can be added to the backlog to be add in the future if desired?

Reply:
Can we use an array-like (but not an array)? Eg,

```js
Object.defineProperty(target, 'files', {
  value: { 0: { ... file0 }, 1: { ... file1 }, length: 2 },
});
```

Reply:
This worked, thanks for the suggestion Justin! 👍